exercises/concept/recycling-robot/.meta/exemplar.js

```diff
@@ -0,0 +1,130 @@
+// @ts-check
+//
+// The line above enables type checking for this file. Various IDEs interpret
+// the @ts-check directive. It will give you helpful autocompletion when
+// implementing this exercise.
+
+export class ElectronicDevice {
+  // This class will be used in the exercise.
+}
```

Comment:
You can import this from another file `lib.js` (and define it in the meta config). See `translation-service` for an example. 

It will prevent people from touching it.